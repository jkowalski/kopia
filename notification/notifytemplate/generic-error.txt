Subject: Kopia has encountered an error during {{ .EventArgs.Operation }} on {{.Hostname}}

Operation: {{ .EventArgs.OperationDetails }}
Started:   {{ .EventArgs.StartTimestamp }}
Finished:  {{ .EventArgs.EndTimestamp }} ({{ .EventArgs.Duration }})

{{ .EventArgs.ErrorDetails }}

Generated at {{ .EventTime | formatTime }} by Kopia {{ .KopiaBuildVersion }}.

https://kopia.io/