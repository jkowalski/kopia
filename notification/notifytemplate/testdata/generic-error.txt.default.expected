Subject: Kopia has encountered an error during Some Operation on some-host

Operation: Some Operation Details
Started:   2020-01-02T03:04:05Z
Finished:  2020-01-02T03:04:06Z (1s)

error details

Generated at 2020-01-02T03:04:05Z by Kopia v0-unofficial.

https://kopia.io/